### PR TITLE
Add special handler for CancellationError in window.onerror.

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
@njam please review.

**Comment by vogdb** [comment]
I couldn't reproduce the problem on the dev version and couldn't logically find where it comes from. That is why I've decided to extend the existing handler of unexpected errors.

**Comment by njam** [comment]
@tomaszdurka can you review this?

File: client-vendor/after-body/00-jserror/jserror.js [hunk 1]
```diff
@@ -26,7 +26,10 @@
 			if (window.onerror) {
 				this.onerrorBackup = window.onerror;
 			}
-			window.onerror = function(message, fileUrl, fileLine) {
+			window.onerror = function(message, fileUrl, fileLine, fileColumn, error) {
+        if (error instanceof Promise.CancellationError) {
+          message = '###' + message + ' :' + fileColumn + ' ' + error.stack
+        }
```
Comment (tomaszdurka) [inline client-vendor/after-body/00-jserror/jserror.js]: Formatting.
Why do you format this message in this special way with hashes in front? 
Also error.stack is multiline right? The it's pretty strange first line is with message/filecolumn and other below.
Do you have example message for this?

**Comment by vogdb** [comment]
@tomaszdurka please review.

**Comment by vogdb** [comment]
@tomaszdurka please review.

**Comment by tomaszdurka** [comment]
Well, if we have no other choice....

**Comment by tomaszdurka** [comment]
Looks good!

**Pull request merged**